webpack.config.js

```diff
@@ -47,13 +48,30 @@ module.exports = {
         path: staticPath
     },
     resolve: {
+        alias: { 'monaco-vim': '@apmorton/monaco-vim' },
```

Comment:
I wonder if an alias or equivalent for `monaco` to the `monaco/esm/..../editor.api` above might save on my old brain remembering how to include monaco (and reduce the number of changes in this PR). Assuming that works: I suspect it doesn't cos it's the kind of thing you'd've thought of already :)